requirements.txt

```diff
@@ -0,0 +1,9 @@
+autopep8
```

Comment:
dependencies should be setup in pyproject.yaml, https://github.com/Sceptre/jinja-unquote-resolvers-filter/blob/master/pyproject.toml#L25-L33

would be good to constrain versions as well.